### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -6986,6 +6986,7 @@ informations->information
 informaton->information
 informtion->information
 inforrmation->information
+infos->info
```
Comment (larsoner) [inline codespell_lib/data/dictionary.txt]: `info` itself is a bit informal, and I wouldn't be surprised if `infos` were a commonly used (informal) pluralization of the word

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: for the record: we do have `informations->informations`. Though I do agree with you, I'm not hardlined about it either way.

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: reverted in 36a825f

**Comment by luzpaz** [comment]
PR should be good to go

**Pull request merged**